pkg/fetcher/fetcher.go

```diff
@@ -611,12 +611,16 @@ func (fetcher *Fetcher) archive(src string, dst string) error {
 	} else {
 		files = append(files, src)
 	}
-	return archiver.DefaultZip.Archive(files, dst)
+	zip := archiver.NewZip()
+	defer zip.Close()
```

Comment:
DeafultZip does not seem to be the issue.
It's the `defer` call that's fixed our issue. We missed to close the archive.

Reply:
Is that so? I thought you were using DefaultZip as the only global obj. This is your design.

Reply:
If there are many requests per second and DefaultZip has not been closed, the problem still exists.

Reply:
Yes, In both cases DefaultZip and NewZip, archive should be closed.